features/karaf/src/main/feature/feature.xml

```diff
@@ -55,7 +55,6 @@
     <bundle>mvn:org.eclipse.smarthome.io/org.eclipse.smarthome.io.monitor/${project.version}</bundle>
     <bundle>mvn:org.eclipse.smarthome.io/org.eclipse.smarthome.io.net/${project.version}</bundle>
     <bundle>mvn:org.eclipse.smarthome.io/org.eclipse.smarthome.io.rest/${project.version}</bundle>
-    <bundle>mvn:org.eclipse.smarthome.io/org.eclipse.smarthome.io.rest.auth/${project.version}</bundle>
```

Comment:
I am not a fan of that optional imports.
It makes the stuff much more difficult than it should be.
Some time someone will change the code and use an optional import and we will run into NCDF exceptions at runtime etc.
I would like to keep the separate bundle and add it to the resp. features as it is done in this PR.